includes/fbbot/class-omise-fbbot-request-handler.php

```diff
@@ -0,0 +1,157 @@
+<?php
+defined( 'ABSPATH' ) or die( "No direct script access allowed." );
+
+if ( class_exists( 'Omise_FBBot_Request_Handler' ) ) {
+	return;
+}
+
+class Omise_FBBot_Request_Handler {
+	
+	private function __construct() {
+		// Hide the constructor
+	}
+
+	public static function handle_message_from( $sender_id, $user_message ) {
+		$message_store = Omise_FBBot_Message_Store;
+		$message = strtolower( $user_message );
+
+		if ( $message_store::check_greeting_words( $message ) ) {
```

Comment:
@igroomgrim also, wouldn't it be more easier if `Omise_FBBot_Conversation_Generator` is the one who check and determine how to answer by itself?

Like,
```php
$bot     = Omise_FBBot_Conversation_Generator::listen( $message );
$message = $bot->reply();
```

Then you don't need to go back n forth between 2 classes (or more, if you have more places to handle response message in the near future). Instead, you just tell your `Bot` class to listen, then reply.